ports/unix/modos.c

```diff
@@ -3,8 +3,8 @@
  *
  * The MIT License (MIT)
  *
+ * Copyright (c) 2014-2018 Paul Sokolovsky
  * Copyright (c) 2013, 2014 Damien P. George
- * Copyright (c) 2014 Paul Sokolovsky
```

Comment:
Why is the ordered changed here (and elsewhere)?  Since 2013 is earlier than 2014 it makes sense to retain the order.

Reply:
I changed order in files which were initially created by me. I hope you would agree that it makes sense if the original author were listed first.

Also, arguably, your copyright year would be adjusted to match the real history of a file, and if you remember, I did that once in a previous patchset on this matter, but refrained from doing this again, as ultimately, you're the best party to maintain your copyright notice.

(Finally, many copyright notices in files contributed by me in early period get your copyright mostly by cut-n-paste, with rationalization that while I create some module from scratch, the structure of it is governed (and copied) from your substantial effort. While later it became clear that it doesn't scale to have any MicroPython-related file ever written to include your copyright, I obviously don't try to revision notices added at that early time.)

Reply:
Ok, you're right that this file looks to be completely fresh (or at least fresh enough) at its time of creation.  So I should adjust my year to match the history.